packages/resin-mixpanel-client/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+* Mixpanel users will be created if they do not exist on `login`/`signup` in Node (this previously only happened in the browser).
+* Mixpanel is now bundled, and loaded synchronously, so should never fail to load.
+* **Breaking!** `set` and `setOnce` have now been removed.
```

Comment:
This is because they were never used anywhere, and you can't easily make them work consistently between Node (which refuses to keep any state) and the browser (which does).